package.json

```diff
@@ -39,7 +39,7 @@
     "@babel/core": "^7.22.11",
     "@khanacademy/flow-to-ts": "^0.5.2",
     "@napi-rs/cli": "^2.15.2",
-    "@parcel/babel-register": "*",
+    "@parcel/babel-register": "2.10.1",
```

Comment:
moved change to a [separate PR](https://github.com/parcel-bundler/parcel/pull/9338) for more flexibility if want to solve this issue with a different approach